lib/command/deploy_image.rb

```diff
@@ -5,18 +5,23 @@ class DeployImage < Base
     NAME = "deploy-image"
     OPTIONS = [
       app_option(required: true),
-      run_release_phase_option
+      run_release_phase_option,
+      skip_pre_deploy_hook_option
     ].freeze
-    DESCRIPTION = "Deploys the latest image to app workloads, and runs a release script (optional)"
+    DESCRIPTION = "Deploys the latest image to app workloads, and runs a pre-deploy hook (optional)"
     LONG_DESCRIPTION = <<~DESC
       - Deploys the latest image to app workloads
-      - Optionally runs a release script before deploying if specified through `release_script` in the `.controlplane/controlplane.yml` file and `--run-release-phase` is provided
-      - The release script is run in the context of `cpl run` with the latest image
-      - The deploy will fail if the release script exits with a non-zero code or doesn't exist
+      - Optionally runs a pre-deploy hook before the image is deployed,
+        which can be either a relative path (to the `.controlplane/` directory) for a script or a command
+        in the context of `cpl run` and with the latest image
+        (if specified through `pre_deploy_hook` in the `.controlplane/controlplane.yml` file)
+      - For backward compatibility, `--skip-pre-deploy-hook` defaults to `true`,
+        and `--run-release-phase` means the same as `--no-skip-pre-deploy-hook`
+      - The deploy will fail if the hook exits with a non-zero code
     DESC
 
-    def call # rubocop:disable Metrics/MethodLength
-      run_release_script if config.options[:run_release_phase]
+    def call # rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLength
```

Comment:
I made sure that changes here are backwards compatible.

`deploy-image` will skip the pre-deploy hook by default, but force it to run with `--run-release-phase` or `--no-skip-pre-deploy-hook`.